code/modules/mining/equipment_locker.dm

```diff
@@ -556,14 +556,11 @@
 				L.Weaken(3)
 				if(ishuman(L))
 					shake_camera(L, 20, 1)
+					var/mob/living/carbon/human/H = L
+					H << "<span class='danger'>As you emerge from the tunnel, your head spins as you lose your footing, and you flop to the ground</span>"
 					spawn(20)
-						if(L)
```

Comment:
Why not just add the mouthcheck here and leave everything else intact? That way we can leave fakevomit as a "can be used on anyone" for admin shennanigans ("B.A.R.F pukes all over himself!").